R/deployApp.R

```diff
@@ -258,6 +259,15 @@ deployApp <- function(appDir = getwd(),
     }
   }
 
+  check_character(envVars, allow_null = TRUE)
+  if (!is.null(envVars) && any(nzchar(names2(envVars)))) {
```

Comment:
I think this is sufficient:

```suggestion
  if (!is.null(envVars) && !is.null(names(envVars))) {
```